docs/modules/plugins/pages/plugin-email.adoc

```diff
@@ -0,0 +1,148 @@
+= Plugin Email
+
+== General Info
+
+The plugin provides functionality interact with email servers over IMAPS protocol.
+
+== Installation
+
+.build.gradle
+[source, gradle]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-email', version: '<vividusVersion>')
+----
+
+== Properties
+
+=== Required
+
+* `email.server.<configuration>.username` - defines email server username
+* `email.server.<configuration>.password` - defines email server password
+* `email.server.<configuration>.properties.host` - defines email server host, e.g. `imap.gmail.com`
+* `email.server.<configuration>.properties.port` - defines email server port, e.g. `993`
+
+=== Optional
+
+* `email.imap.wait.duration` - defines message polling duration from the email server, default value is `PT30S`
+* `email.imap.wait.retry-times` - defines a number of retries will be performed during polling duration, default value is `5`
+* `email.imap.folder` - defines folder name to work with, default value is `Inbox`
+
+Polling timeout that can be calculated by `email.imap.wait.duration / email.imap.wait.retry-times` must be equal to or exceed `5` seconds.
```

Comment:
is it really important here?

Reply:
imho yes, I would prefer to save end users time on re-runs by documenting these rules

Reply:
`be equal to or exceed` -> `be not less than`